### PR TITLE
samples: fs: littlefs: make sample workable for mmc disk driver

### PR DESCRIPTION
**Pull request by xakep-amatop**
The LittleFS sample may be used for both SDMMC and MMC disk drivers, so make it possible to work with different volume disk driver names.

### CODE REVIEW

File: samples/subsys/fs/littlefs/README.rst [hunk 1]
```diff
@@ -88,7 +88,7 @@ To build the test:
    :gen-args: -DCONF_FILE=prj_blk.conf
    :compact:
 
-One can also set ``CONFIG_SDMMC_VOLUME_NAME`` to provide the mount point name
+One can also set ``CONFIG_SDMMC_VOLUME_NAME`` or ``CONFIG_MMC_VOLUME_NAME`` (in case of MMC) to provide the mount point name
```
Comment (de-nordic) [inline samples/subsys/fs/littlefs/README.rst:91]: I think we should reword this to make it clear that, depending on selected device, the name of device will be used form mount point and will be take from one of above configs. Somehow I do not like the "One can also set (...)" beginning of sentence and would rather have "Mount point is controlled by"

**Review by de-nordic — CHANGES_REQUESTED** [review]
Single comment to documentation, otherwise looks OK to me.

File: samples/subsys/fs/littlefs/README.rst (hunk 1 above)
Comment (xakep-amatop) [reply samples/subsys/fs/littlefs/README.rst]: got it, I have rewritten this part of the littlefs sample documentation
thank you for a review

**Review by de-nordic — APPROVED** [review]

**Review by Laczen — APPROVED** [review]
